ansible/group_vars/delfin.yml

```diff
@@ -36,6 +36,10 @@ venv: "{{ delfin_work_dir }}/venv"
 delfin_redis_ip: 127.0.0.1
 delfin_redis_port: 26379
 
+delfin_rabbitmq_user: delfinuser
+delfin_rabbitmq_pass: delfinpass
+
```

Comment:
Line 52 - > metrics_cache_file = /var/lib/delfin/delfin_exporter.txt is not required now, instead we need metrics_dir = '/var/lib/delfin/metrics'
here .

Reply:
Also we need to add option for perf ocllection interval config 
[TELEMETRY]
performance_collection_interval = 900

Reply:
Thanks, updated